editor/babel.config.js

```diff
@@ -0,0 +1,13 @@
+// We only use this for Jest for now
+
+module.exports = (api) => {
```

Comment:
I add a babel.config but the tests are not yet using babel-jest. that will come in a later PR